extensions/wikia/ContentReview/services/ContentReviewService.class.php

```diff
@@ -83,4 +84,53 @@ public function deletePageData( $wikiId, $pageId ) {
 			] );
 		}
 	}
+
+	public function escalateReview( $wikiId, $pageId, $diff, $oldid ) {
+		$title = \GlobalTitle::newFromId( $pageId, $wikiId );
+		if ( !$title instanceof \GlobalTitle ) {
+			return false;
+		}
+
+		if ( $diff === 0 ) {
+			$revisionId = $oldid;
+		} else {
+			$revisionId = $diff;
+		}
+
+		$reviewModel = new ReviewModel();
+		$sqlStatus = $reviewModel->escalateReview( $wikiId, $pageId, $revisionId );
+
+		if ( !$sqlStatus ) {
+			return false;
+		} else {
```

Comment:
Since there is a return in the `if`, this `else` block is unnecessary and the code can be unindented.